plugins/Login/templates/loginLayout.twig

```diff
@@ -9,12 +9,16 @@
     {{ parent() }}
 {% endblock %}
 
-{% set title %}{{ 'Login_LogIn'|translate }}{% endset %}
+
+{% set title %}{{ pageTitle|default('Login_LogIn'|translate) }}{% endset %}
 
 {% block pageDescription %}{{ 'General_OpenSourceWebAnalytics'|translate }}{% endblock %}
 
 {% set bodyId = 'loginPage' %}
 
+{% set centeredLogo = isCenteredLogo is defined ? isCenteredLogo : true %}
```

Comment:
That one is actually not needed, as the variable is only used once and could be simplified with a '??` usage
```suggestion
```

Reply:
@sgiehl Updated as suggested